projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -5731,6 +5731,12 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
      * @internal
      */
     public copyHandler(event) {
+        const eventPathElements = event.composedPath().map(el => el.tagName?.toLowerCase());
```

Comment:
12.3 still supports IE11 and per https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath , this method is not available in IE. You need to ensure it will not throw.